### PR TITLE
nimrod monkey patch

### PR DESCRIPTION
**Pull request by TomekTrzeciak**
Enable reading nimrod radar files with iris while waiting for scitools/iris#3647 to make it through the pipeline.

Testing:
 - [x] Ran some tests locally by calling standardise on radar precip and coverage files.

### CODE REVIEW

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Complexity increasing per file
==============================
- improver/utilities/load.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5023279&bid=16556356)

File: improver/utilities/load.py [hunk 1]
```diff
@@ -38,6 +39,16 @@
     enforce_coordinate_ordering, merge_cubes)
 
 
+@contextlib.contextmanager
+def monkeypatched(object, name, patch):
```
Comment (benfitzpatrick) [inline improver/utilities/load.py]: Codacy Issue found: [Redefining built-in 'object'](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5023279)

**Comment by benfitzpatrick** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
- Added 1
           

Complexity increasing per file
==============================
- improver/utilities/load.py  6
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5023279&bid=16556356)

**Comment by TomekTrzeciak** [comment]
> nice contextmanaged monkeypatching :)

I was still working on this...

**Comment by TomekTrzeciak** [comment]
> > nice contextmanaged monkeypatching :)
> 
> I was still working on this...

We need some way of blocking PRs from merge while being worked on. It's not the first time this happens to me.

**Pull request merged**